client/my-sites/theme/main.jsx

```diff
@@ -839,6 +841,7 @@ export default connect(
 			canUserUploadThemes: hasFeature( state, siteId, FEATURE_UPLOAD_THEMES ),
 			// No siteId specified since we want the *canonical* URL :-)
 			canonicalUrl: 'https://wordpress.com' + getThemeDetailsUrl( state, id ),
+			demoUrl: getThemeDemoUrl( state, id ),
```

Comment:
Isn't siteId the third argument here?

Or is this similar to the above property, where we are going for the "canonical demo URL" ?

Reply:
I noticed when we get the `theme` variable above, we use:

```
const theme = getCanonicalTheme( state, siteId, id );
```

getThemeDemoUrl uses getCanonicalTheme under the hood

```
export function getThemeDemoUrl( state, themeId, siteId ) {
	const theme = getCanonicalTheme( state, siteId, themeId );
```

So I think the usage of getCanonicalThemeUrl should be the same in both places, since it's the same connector function?